health.go

```diff
@@ -80,6 +81,7 @@ func NewDefaultHealthCheck() *HealthCheck {
 		GracePeriodSeconds:     30,
 		IntervalSeconds:        10,
 		TimeoutSeconds:         5,
+		DelaySeconds:           15,
```

Comment:
I think Marathon API's default value of 15 which I give as a default in NewDefaultHealthCheck() should be allright.